Phase02/FullTextSearch/Logic/EmptyRemover.cs

```diff
@@ -0,0 +1,11 @@
+namespace FullTextSearch;
+
+public static class EmptyRemover
+{
+    public static List<string> RemoveEmptyCells(this List<string> list)
+    {
+        return (from word in list
+            where word != string.Empty
+            select word).ToList();
```

Comment:
LINQ to Objects instead of SQL
https://xerostory.com/8-types-of-linq-query-in-csharp/